### ISSUE TITLE
Internal tracking

### ISSUE THREAD

**Issue by scottalreadytaken** [pluginkollektiv/statify#118]
I would be using this plugin if it could help me understand how people move around the website using the internal referrer. Unless I’m mistaken the plugin currently filters out the referrer field if it’s an internal link. Would anyone else be interested in being able to review internal referrers? Perhaps as a separate view?

**Comment by Zodiac1978** [comment]
Looks like a good idea. At least as an option. We need to discuss this further. @stklcode What do you think?

**Comment by stklcode** [comment]
> Unless I’m mistaken the plugin currently filters out the referrer field if it’s an internal link. 

That's right Referrers is only added if not prefixed with the current site's home URL:
https://github.com/pluginkollektiv/statify/blob/f7985e3dba345415f7ddb9542557079a9dedc74e/inc/class-statify-frontend.php#L87-L89)

Of course we can simply omit this constraint. Immediate result would be a messed up referrer list. Filtering the top lists for larger datasets is not very efficient.

We could add another DB column like `referrer_int` to simply separate both lists without performance issues. Probably parse to relative paths for automatic grouping on multi-domain sites.

The larger issue imho is the visualization. What comes to my mind is an "internal/external" switch above the existing referrer list. This however does not show the navigation paths - which are much more interesting than simply identifying local hubs. Using such UI switch to replace the current 2 tables with one large table and group not only by referrer but by path could fit the scope.

**External view:**
```text
Top referers:    Top targets:
50 example.com   80 /
40 example.net   70 /foo/
30 example.org   60 /bar/
```

**Internal view:**
```text
Top paths:
60 /     -> /foo/
50 /foo/ -> /bar/
40 /bar/ -> /baz/
```
